### ISSUE TITLE
Unmet dependencies using 0.8.11 .deb on Ubuntu 22.04

### ISSUE THREAD

**Issue by SoerenBusse** [rtbrick/bngblaster#145]
**Describe the bug**

Installing BNGBlaster 0.8.11 .deb package on Ubuntu 22.04 leads to unment dependencies with libssl1.1.

**To Reproduce**

```
wget https://github.com/rtbrick/bngblaster/releases/download/0.8.11/bngblaster-0.8.11-ubuntu-22.04_amd64.deb

# Shows libssl1.1 as dependency which is deprecated since Ubuntu 22.04
dpkg -I bngblaster-0.8.11-ubuntu-22.04_amd64.deb

 neues Debian-Paket, Version 2.0.
 Größe 471814 Byte: control-Archiv= 469 Byte.
     283 Byte,    11 Zeilen      control
     266 Byte,     5 Zeilen      md5sums
 Architecture: amd64
 Depends: libssl1.1, libncurses5, libjansson4
 Description: RtBrick BNG Blaster
 Homepage: https://github.com/rtbrick/bngblaster
 Maintainer: RtBrick <bngblaster@rtbrick.com>
 Package: bngblaster
 Priority: optional
 Section: devel
 Version: 0.8.11
 Installed-Size: 1085
```

This leads to unmet dependencies when using apt to install the package:
```
~# apt install ./bngblaster-0.8.11-ubuntu-22.04_amd64.deb
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Note, selecting 'bngblaster' instead of './bngblaster-0.8.11-ubuntu-22.04_amd64.deb'
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 bngblaster : Depends: libssl1.1 but it is not installable
E: Unable to correct problems, you have held broken packages.
```

**Expected behavior**

Ubuntu 22.04 should have libssl3 as dependency

**Comment by GIC-de** [comment]
Good catch, will include this fix in the next release!!!

**Comment by GIC-de** [comment]
As workaround, you can install dependancies manually and install BNG Blaster with the command `dpkg -i bngblaster-0.8.11-ubuntu-22.04_amd64.deb`.

**Comment by nitzan-tz** [comment]
To install these dependencies use the following
```
echo "deb http://security.ubuntu.com/ubuntu focal-security main" | sudo tee /etc/apt/sources.list.d/focal-security.list

sudo apt-get update
sudo apt-get install libssl1.1 libncurses5
```

Then install bngbluster

**Comment by GIC-de** [comment]
Fixed with 0.8.12:

```
$ dpkg -I bngblaster-0.8.12-ubuntu-18.04_amd64.deb
 new Debian package, version 2.0.
 size 443206 bytes: control archive=469 bytes.
     282 bytes,    11 lines      control
     266 bytes,     5 lines      md5sums
 Architecture: amd64
 Depends: libssl1.1, libncurses5, libjansson4
 Description: RtBrick BNG Blaster
 Homepage: https://github.com/rtbrick/bngblaster
 Maintainer: RtBrick <bngblaster@rtbrick.com>
 Package: bngblaster
 Priority: optional
 Section: devel
 Version: 0.8.12
 Installed-Size: 990

$ dpkg -I bngblaster-0.8.12-ubuntu-20.04_amd64.deb
 new Debian package, version 2.0.
 size 477532 bytes: control archive=470 bytes.
     283 bytes,    11 lines      control
     266 bytes,     5 lines      md5sums
 Architecture: amd64
 Depends: libssl1.1, libncurses5, libjansson4
 Description: RtBrick BNG Blaster
 Homepage: https://github.com/rtbrick/bngblaster
 Maintainer: RtBrick <bngblaster@rtbrick.com>
 Package: bngblaster
 Priority: optional
 Section: devel
 Version: 0.8.12
 Installed-Size: 1094

$ dpkg -I bngblaster-0.8.12-ubuntu-22.04_amd64.deb
 new Debian package, version 2.0.
 size 473244 bytes: control archive=467 bytes.
     281 bytes,    11 lines      control
     266 bytes,     5 lines      md5sums
 Architecture: amd64
 Depends: libssl3, libncurses6, libjansson4
 Description: RtBrick BNG Blaster
 Homepage: https://github.com/rtbrick/bngblaster
 Maintainer: RtBrick <bngblaster@rtbrick.com>
 Package: bngblaster
 Priority: optional
 Section: devel
 Version: 0.8.12
 Installed-Size: 1085
 ```
